docs/Custom_Operation_for_GPUs.md

```diff
@@ -1,11 +1,12 @@
-# Custom operations for GPUs
+# Custom operations for GPUs with C++ and CUDA
+
+JAX, a popular framework for machine learning, comes with a large number of operations. However, users occasionally run into a situation where they need a new operation that is not supported by JAX.
 
-JAX ships with a large number of built-in operations, but users occasionally run into a situation where they need a new operation that is not supported by JAX.
```

Comment:
It looks like this reverts the change from https://github.com/google/jax/pull/13950 – is that intentional? I like the rephrased intro sentence better (we're in the JAX documentation, we don't need to tell people that JAX is a popular framework for machine learning!)

Reply:
Fixed. Thanks @jakevdp PTAL